### PR TITLE
"make new-repo" for openshift/cloud-network-config-controller

### PR DESCRIPTION
**Pull request by alexanderConstantinescu**
This initializes prow CI for https://github.com/openshift/cloud-network-config-controller

This should not merge until https://github.com/openshift/cloud-network-config-controller/pull/1 does, so

/hold

Not sure if I can do this, but it would be nice:

/cc @openshift/openshift-team-sdn

### CODE REVIEW

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -3240,6 +3244,9 @@ lgtm:
   - openshift-priv/azure-disk-csi-driver
   - openshift/azure-disk-csi-driver
   review_acts_as_lgtm: true
+- repos:
+  - openshift/cloud-network-config-controller
+  review_acts_as_lgtm: true
```
Comment (abhat) [inline core-services/prow/02_config/_plugins.yaml:3249]: what is the implication of this tag? If someone finishes the review, does that automatically LGTM the PR?

Comment (alexanderConstantinescu) [reply core-services/prow/02_config/_plugins.yaml:3249]: Not sure, I just ran `make new-repo`... @alvaroaleman , could you fill us in?

**Comment by alexanderConstantinescu** [comment]
We seem to be good now. 

/hold cancel

/assign @abhat @squeed 

Since I needed to add the `OWNERS` files manually in `config/` and `jobs/`, and since all other `OWNERS` files in this repo have  the following specified (https://github.com/openshift/release/blob/master/ci-operator/config/openshift/egress-router-cni/OWNERS#L1):

```
# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
```

I have doubts as to if `make new-repo` performed the entire setup correctly. Hence I'd appreciate a quick review from someone knowledgeable

/cc @stevekuznetsov

File: core-services/prow/02_config/_plugins.yaml (hunk 1 above)
Comment (alvaroaleman) [reply core-services/prow/02_config/_plugins.yaml:3283]: It means that the github review (when you click review -> approve) will count as lgtm

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**